tsconfig.json

```diff
@@ -7,12 +7,13 @@
     "incremental": false,
     "isolatedModules": true,
     "jsx": "react",
+    "moduleResolution": "bundler",
     "lib": ["es2020", "dom"],
-    "module": "commonjs",
+    "module": "Preserve",
     "noImplicitAny": true,
     "rootDir": "./src",
     "skipLibCheck": true,
-    "target": "ES2020"
+    "target": "ES2022"
```

Comment:
@ndelangen are these worth communicating to existing addon authors as additional updates to make on their addons?

Reply:
Good callout, but I don't think this is that important for addon authors to upgrade.

I do think we should align it with Node 20 though: https://github.com/microsoft/TypeScript/wiki/Node-Target-Mapping#node-20